doc/110_talks.rst

```diff
@@ -17,6 +17,8 @@ Talks
 
 The following talks will be or have been given about restic:
 
+-  2021-03-21: `The Changelog: Restic has your backup 
```

Comment:
Where did you get this date from? According to the Changelog page for this podcast it was recorded on the 10th of March and published no the 2nd of April, neither of which match :)

Reply:
As listed in the issue https://github.com/restic/restic/issues/4263 that i created I took it litterally from 
https://restic.net/#talks where it prints:
2021-03-21: [The Changelog: Restic has your backup (Podcast)](https://changelog.com/podcast/434)


And you are right, the podcast page https://changelog.com/podcast/434 mentions other dates. I can only guess above date of 21st March is when the restic.net page was updated.

I would propose to make it the date it was published as that is what you see in the podcast list and podcast app, being 2nd of April 2021.
https://changelog.com/podcast?page=5#feed

And I would propose to align the https://restic.net/#talks date for this entry.


Can you agree?

Reply:
Ah, that makes sense. Yeah, that is the date that the page on the website was published. I agree with you, better to use the 2nd of April 2021 as the date in your PR :) Can you make that change and amend/squash the commit?

I can align the date, doesn't hurt.

Reply:
You did!